views/sass/_custom.scss

```diff
@@ -431,3 +431,7 @@ table.definition {
   cursor: default;
 }
 
+.download-terms__highlighted {
+  background-color: #fffadd;
```

Comment:
Any chance you could make this colour a Sass variable, in `_variables.scss`. Maybe something like `$colour_pale_yellow`?